### ISSUE TITLE
Latest official version of Weather Station firmware has different header

### ISSUE THREAD

**Issue by GotenXiao** [Scrin/RuuviCollector#1]
The latest version of the Weather Station DFU image has a single byte difference in the header to the sensortag firmware:
```java
    private static final String SENSORTAG_BEGINS = "> 04 3E 25 02 01 03 01 ";
```

I'm not sure exactly what difference in the DFU image causes this change in the packet, but I've now got a working Grafana dashboard looking at an InfluxDB data source with three tags logging successfully after changing this string.

**Comment by Scrin** [comment]
If I remember correctly, the third byte in the raw packet is the packet length.. I wonder what has changed.. Thanks for notifying, I will look at this soon.

**Comment by Scrin** [comment]
Now fixed in version [v0.1.1](https://github.com/Scrin/RuuviCollector/releases/tag/v0.1.1-beta)
